### PR TITLE
add sarah and scottish summit

### PR DESCRIPTION
**Pull request by weeyin83**
Adding details for myself and for the Scottish Summit conference

### CODE REVIEW

File: data/locations.json [hunk 1]
```diff
@@ -589,6 +589,7 @@
             "Aberdeen": "1",
             "Crieff": "1",
             "Edinburgh": "1"
```
Comment (iiqrah) [inline data/locations.json]: a ',' needs to be added after "Edinburgh": "1" to add the location correctly.

**Comment by szabgab** [comment]
Hi Sarah, 
thanks for the submission.
it is a bit awkward, but the "state" should be "Scotland" for Glasgow.

**Comment by szabgab** [comment]
Excellent. Thank you!

**Comment by szabgab** [comment]
published at:
https://codeandtalk.com/e/scottish-summit-2020
https://codeandtalk.com/p/sarah-lean

**Pull request merged**